electronic_store/data/ir_actions_act_window.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo>
+    <record id="x_project_task_worksheet_template_1_ir_actions_act_window_1" model="ir.actions.act_window">
```

Comment:
Where is it used?

Reply:
It is passed as reference in action_id of worksheet template file